docs/services/aws-services-guide.md

```diff
@@ -14,11 +11,9 @@ Crossplane managed resources and the official Wordpress Docker image.
 
 1. [Pre-requisites](#pre-requisites)
 1. [Preparation](#preparation)
-1. [Set Up an EKS Cluster](#set-up-an-eks-cluster)
-1. [Set Up RDS Configurations](#set-up-rds-configurations)
-1. [Set Up Crossplane](#set-up-crossplane)
+1. [Create Configuration](#create-configuration)
```

Comment:
I like the simpler ToC, and shorter doc in general, it's a big improvement!

Suggest the following ToC:

1. Pre-requisites
1. Preparation
1. Set Up Crossplane
    1. Install
    1. Cloud Provider
    1. Resource Classes
    1. Configure Managed Service Access (DBSubnetGroup, SecurityGroup)
1. Provision MySQL
   1. Resource Claim
1. Install Wordpress
1. Clean Up
1. Conclusion and Next Steps